crypto/src/main/scala/coop/rchain/crypto/signatures/Secp256k1.scala

```diff
@@ -37,7 +37,9 @@ object Secp256k1 {
     kpg.initialize(new ECGenParameterSpec(curveName), SecureRandomUtil.secureRandomNonBlocking)
     val kp = kpg.generateKeyPair
 
-    val sec = Base16.decode(kp.getPrivate.asInstanceOf[ECPrivateKey].getS.toString(16))
+    val padded = org.apache.commons.lang3.StringUtils
+      .leftPad(kp.getPrivate.asInstanceOf[ECPrivateKey].getS.toString(16), 64, '0')
```

Comment:
I guess since we have guava on classpath, you should use `Strings.padStart` and not add the apache-commons dependency.

I'd say that even if I didn't hate apache commons :)